content/en/docs/reference/access-authn-authz/rbac.md

```diff
@@ -70,55 +98,72 @@ metadata:
   name: secret-reader
 rules:
 - apiGroups: [""]
+  #
+  # at the HTTP level, the name of the resource for accessing Secret
+  # objects is "secrets"
   resources: ["secrets"]
   verbs: ["get", "watch", "list"]
 ```
 
 ### RoleBinding and ClusterRoleBinding
 
 A role binding grants the permissions defined in a role to a user or set of users.
-It holds a list of subjects (users, groups, or service accounts), and a reference to the role being granted.
-Permissions can be granted within a namespace with a `RoleBinding`, or cluster-wide with a `ClusterRoleBinding`.
+It holds a list of *subjects* (users, groups, or service accounts), and a reference to the
+role being granted.
+A RoleBinding grants permissions within a specific namespace whereas a ClusterRoleBinding
+grants that access cluster-wide.
 
-A `RoleBinding` may reference a `Role` in the same namespace.
-The following `RoleBinding` grants the "pod-reader" role to the user "jane" within the "default" namespace.
-This allows "jane" to read pods in the "default" namespace.
+A RoleBinding may reference any Role in the same namespace. Alternatively, a RoleBinding
+can reference a ClusterRole and bind that ClusterRole to the namespace of the RoleBinding.
+If you want to bind a ClusterRole to all the namespaces in your cluster, you use a
+ClusterRoleBinding.
+
+#### RoleBinding examples #{rolebinding-example}
```

Comment:
@sftim , double check: {#rolebinding-example}

Reply:
Fixed this one. Oops.